src/service/QuestionService.js

```diff
@@ -0,0 +1,44 @@
+import {
+    getPokemonById
+} from "../api/pokemons";
+import { WHAT_DOES_THIS_POKEMON_LOOK_LIKE, WHO_IS_THAT_POKEMON } from "./modes";
+
+export class QuestionService {
+
+    constructor(mode) {
+        this.mode = mode;
+        this.correctAnswerIndex = 0; // range 0-3
```

Comment:
I see what you did there! It is **ok** so far, but it will produce some complications later, when you want to randomise order of answers.

But it will be in a new task :)

Reply:
Ok, I will change it in new task.